api/saucectl.schema.json

```diff
@@ -2485,7 +2485,7 @@
                     },
                     "formatOptions": {
```

Comment:
Where in the playwright(-cucumber) runner is this setting being consumed? I can't seem to find any references 👁️

Reply:
In my cucumber PR, I fixed it https://github.com/saucelabs/sauce-playwright-runner/pull/284/files#diff-505cf791157e41362aa4dd9f4118eeef459e6a50a6bd07be8d6d04fbabf03919R207